installer/install.iss

```diff
@@ -2058,28 +2085,28 @@ begin
     end;
 
     (*
-     * Create a custom page for using (Tortoise)Plink instead of OpenSSH
-     * if at least one PuTTY session is found in the Registry.
+     * Create a custom page for using (Tortoise)Plink or a self-supplied OpenSSH instead of bundled OpenSSH
+     * if at least one PuTTY session is found in the Registry or an OpenSSH binary is found on the PATH.
      *)
 
-    if RegGetSubkeyNames(HKEY_CURRENT_USER,'Software\SimonTatham\PuTTY\Sessions',PuTTYSessions) and (GetArrayLength(PuTTYSessions)>0) then begin
-        PuTTYPage:=CreatePage(PrevPageID,'Choosing the SSH executable','Which Secure Shell client program would you like Git to use?',TabOrder,Top,Left);
+    if (RegGetSubkeyNames(HKEY_CURRENT_USER,'Software\SimonTatham\PuTTY\Sessions',PuTTYSessions) and (GetArrayLength(PuTTYSessions)>0)) or
+       (FileSearch('ssh.exe', GetEnv('PATH')) <> '') then begin
+        SSHChoicePage:=CreatePage(PrevPageID,'Choosing the SSH executable','Which Secure Shell client program would you like Git to use?',TabOrder,Top,Left);
 
         // 1st choice
-        RdbSSH[GS_OpenSSH]:=CreateRadioButton(PuTTYPage,'Use OpenSSH','This uses ssh.exe that comes with Git. The GIT_SSH and SVN_SSH'+#13+'environment variables will not be modified.',TabOrder,Top,Left);
+        RdbSSH[GS_OpenSSH]:=CreateRadioButton(SSHChoicePage,'Use bundled OpenSSH',
+            'This uses ssh.exe that comes with Git.',
+            TabOrder,Top,Left);
 
         // 2nd choice
-        RdbSSH[GS_Plink]:=CreateRadioButton(PuTTYPage,'Use (Tortoise)Plink',
-            'PuTTY sessions were found in your Registry. You may specify the path'+#13+
-            'to an existing copy of (Tortoise)Plink.exe from the TortoiseGit/SVN/CVS'+#13+
-            'or PuTTY applications. "ssh.variant" will be set in the GIT configuration. '+#13+
-            'The GIT_SSH and SVN_SSH environment variables will be adjusted to point '+#13+
-            'to the following executable:',TabOrder,Top,Left);
-        EdtPlink:=TEdit.Create(PuTTYPage);
+        RdbSSH[GS_Plink]:=CreateRadioButton(SSHChoicePage,'Use (Tortoise)Plink',
+            'To use PuTTY, specify the path to an existing copy of (Tortoise)Plink.exe:',
+            TabOrder,Top,Left);
```

Comment:
Since this changes the PuTTY text, which is independent of the change that adds a third option, I would like to encourage you to refactor this change out into its own commit.

Reply:
Is a separate commit okay or should I put it into a separate pull request?

Reply:
A separate commit in the same PR is totally okay.